### PR TITLE
Feature/token page

### PR DESCRIPTION
**Pull request by pingustar**

### CODE REVIEW

**Comment by ashachaf** [comment]
- [ ] tool tips:
- liquidity: "The value of tokens staked in the pool"
- remove all other tool tips

- [ ] ticker 
- rotation should be infinite. meaning, always move in the same direction and loop them (first->last->first->last)
- on mouse over, add 2 arrows on the sides (example: https://www.w3schools.com/howto/howto_js_slideshow.asp). when clicking the right arrow, the ticker should move to the left exposing the next right most image.
- on mouse over, enable that moving the scroll wheel in the mouse would move the ticker right or left. scroll up should move the ticker to the right.

File: tailwind.config.js [hunk 1]
```diff
@@ -42,14 +42,14 @@ module.exports = {
...
-        DEFAULT: '#0ED3B0',
+        DEFAULT: '#0FC7A6',
         50: '#f3fdfb',
         100: '#e7fbf7',
         200: '#c3f4eb',
         300: '#9feddf',
         400: '#56e0c8',
         500: '#0ED3B0',
-        600: '#0dbe9e',
+        600: '#0FC7A6',
```
Comment (pingustar) [inline tailwind.config.js:52]: @Ran73Cohen just FYI - I'll fix it but just for future reference, the default is 500

**Comment by Meytal2021** [comment]
- [ ] Please update hover for row in the table (light and dark mode)
https://www.figma.com/file/fwADI9wqDrRAdlMX8EddCw/Bancor-v3?node-id=4588%3A84151

- [ ] Please change the color of i icon to  grey (like the text)
image

**Comment by ashachaf** [comment]
- [ ] shield icon is not aligned 
image

- [ ]  Token title at the moment is 16PX, please make it 14px

- [ ] title sorting - first click should sort the table based on the column descending. for example: click price -> sort highest price to lowest price. click 24h volume -> sort highest volume first to lowest volume last.

- [ ] pagination 
- please make the clickable area for the < > a bit bigger. it is pretty difficult to click on them at the moment.
- when clicking the page number, auto select the existing page so if i enter a new number it will override it.

- [ ] dark mode i icon is white instead of gray
image

- [ ] tokens with no protected pool should NOT show the shield
image

- [ ]

**Pull request merged**